core/src/cats/drop_bareos_database.in

```diff
@@ -30,71 +30,34 @@
 # change to root dir
 cd /
 
+. "@scriptdir@"/bareos-config-lib.sh
 
-. @scriptdir@/bareos-config-lib.sh
+db_name="${db_name:-$(get_database_name @db_name@)}"
+db_type="$(get_database_driver_default)"
 
-db_name="${db_name:-`get_database_name @db_name@`}"
-db_user="${db_user:-`get_database_user @db_user@`}"
-default_db_type=`get_database_driver_default`
-working_dir=`get_working_dir`
-
-#
-# See if the first argument is a valid backend name.
-# If so the user overrides the default database backend.
-#
+# Check if some parameters are given, try to detect old usage with db_type
 if [ $# -gt 0 ]; then
```

Comment:
Same as above: Do we want to exit if more than one parameter is set?

Reply:
No we will not exit, but we won't take care of any, nor pass all to database utility.